### DISCUSSION TITLE
Can we have a full release on nuget please?

### DISCUSSION THREAD

**Discussion by mlockett42** [dsbenghe/Novell.Directory.Ldap.NETStandard#247] [General]
According to this answer from the maintainer version 4.0 is good to use

https://github.com/dsbenghe/Novell.Directory.Ldap.NETStandard/issues/153

If so can we please have the status updated on nuget?

**Comment by dsbenghe** [comment]
Is that needed because for whatever reason you can not use a "beta"?

**Reply by TheConstructor** [reply]
It certain looks better, if your dependencies are not marked as pre-release. On the other hand, do you have any thing you want to get into 4.0, are you waiting on feedback on the stability, or what is holding you back?

**Reply by TheConstructor** [reply]
Thank you for releasing 4.0 to NuGet!

**Reply by FinHorsley** [reply]
Another thanks for the 4.0 release 🎉
